api/health/health.go

```diff
@@ -0,0 +1,17 @@
+package health
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/zimash/hrenovina/api"
+)
+
+func HandlerHealth(w http.ResponseWriter, r *http.Request) {
+	log.Printf("%s from %s\n", r.RequestURI, r.RemoteAddr)
+	w.Header().Add("Content-Type", "application/json")
+	if _, err := fmt.Fprint(w, api.JSONPayload{}); err != nil {
```

Comment:
If you want to use a structure than it's better to call it something like HealthResponse and use JSONEncoder + w.Write(). I understand that fmt.Fprint do almost the same but JSONEncoder + w.Write() will be used in future. So the one style is better.
Or just use simple string "{}" here.

Reply:
Used w.Write, without fmt.Fprint(w...).